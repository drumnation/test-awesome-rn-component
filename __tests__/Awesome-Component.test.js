import { Text, View } from 'react-native';
import React from 'react';
import AwesomeComponent from '../components/awesome-component.js';
// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

const tree = renderer.create(
	<AwesomeComponent
	  hello={'Hello world!'} />
).toJSON();

test('Render properly', () => {
	const hello = tree.children[0].children[0];
	expect(hello).toBe('Hello world!');
});

test('Snapshot match', () => {
  expect(tree).toMatchSnapshot();
});
