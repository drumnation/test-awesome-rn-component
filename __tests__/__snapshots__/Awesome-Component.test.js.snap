// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot match 1`] = `
<View>
  <Text
    accessible={true}
    allowFontScaling={true}
    disabled={false}
    ellipsizeMode="tail"
  >
    Hello world!
  </Text>
</View>
`;
